spock-specs/src/test/groovy/org/spockframework/docs/extension/IgnoreIfDocSpec.groovy

```diff
@@ -62,4 +62,30 @@ class IgnoreIfDocSpec extends Specification {
     true
   }
 
+  def "IgnoreIf can be configured to be inherited"() {
+    when:
+    def result = runner.runWithImports (
+      /* tag::example-g[] */"""
+@IgnoreIf(value = { jvm.java8Compatible }, inherited = true)
+abstract class Foo extends Specification {
+}
+
+class Bar extends Foo {
+  def "I won't run on Java 8 and above"() {
+    expect: true
+  }
+}
+""") // end::example-g[]
```

Comment:
Will the text before the start of the comment be included or not?

Reply:
lines with tags won't be included, that is also the reason why the start tag above is positioned as it is.